### ISSUE TITLE
Service is not scanning domains

### ISSUE THREAD

**Issue by diepnt90** [deividgdt/ytadsblocker#17]
Hi,

I follow the guide but the service is not active as below

```
[i] Youtube Ads Blocker will be installed.
        Log file: /var/log/ytadsblocker.log
        Subdomains added to: /etc/pihole/gravity.db

[i] If you move the script to a different place, please execute it again with the option 'install'
[i] Installing the service...OK. Service installed.
[i] Configuring the database: /etc/pihole/gravity.db ...
[i] Searching for googlevideo.com subdomains...
[w] No subdomains to add at the moment.
[i] Deleting temp...OK. Temp deleted.
[✓] Youtube Ads Blocker: INSTALLED...

[i] To start the service execute as follows: systemctl start ytadsblocker
[i] Enabling the service to start it automatically with the OS.OK.
root@raspberrypi:~/ytadsblocker# systemctl start ytadsblocker
root@raspberrypi:~/ytadsblocker# service ytadsblocker status
● ytadsblocker.service - Youtube ads blocker service for Pi-hole
   Loaded: loaded (/lib/systemd/system/ytadsblocker.service; enabled; vendor preset: enabled)
   Active: inactive (dead) since Wed 2021-10-13 10:38:32 BST; 30s ago
  Process: 12207 ExecStart=/root/ytadsblocker/ytadsblocker.sh start (code=exited, status=0/SUCCESS)
  Process: 12217 ExecStop=/root/ytadsblocker/ytadsblocker.sh stop (code=exited, status=0/SUCCESS)
 Main PID: 12207 (code=exited, status=0/SUCCESS)

Oct 13 10:38:31 raspberrypi systemd[1]: Started Youtube ads blocker service for Pi-hole.
Oct 13 10:38:31 raspberrypi ytadsblocker.sh[12207]: That option does not exists. Usage: ./ytadsblocker.sh -a [ install ( -m aggressive
Oct 13 10:38:32 raspberrypi ytadsblocker.sh[12217]: That option does not exists. Usage: ./ytadsblocker.sh -a [ install ( -m aggressive
Oct 13 10:38:32 raspberrypi systemd[1]: ytadsblocker.service: Succeeded.
```

**Comment by diepnt90** [comment]
hmm no more videos were added :(

tail -f /var/log/ytadsblocker.log
[2021-10-13 14:37:14] Adding the subdomain: r4.sn-42u-i5oe7.googlevideo.com
[2021-10-13 14:37:15] Adding the subdomain: r4.sn-42u-i5oer.googlevideo.com
[2021-10-13 14:37:15] Adding the subdomain: r5.sn-42u-i5ol6.googlevideo.com
[2021-10-13 14:37:15] Adding the subdomain: r5.sn-42u-i5old.googlevideo.com
[2021-10-13 14:37:15] Adding the subdomain: r6.sn-42u-i2is.googlevideo.com
[2021-10-13 14:37:15] Adding the subdomain: r6.sn-42u-i5ole.googlevideo.com
[2021-10-13 14:37:15] Adding the subdomain: r6.sn-42u-i5olz.googlevideo.com
[2021-10-13 14:37:15] Adding the subdomain: r7.sn-42u-i5oee.googlevideo.com
[2021-10-13 14:37:15] Adding the subdomain: r8.sn-42u-i5oe7.googlevideo.com
[2021-10-13 14:37:42] Youtube Ads Blocker has been installed. Welcome!

So I uninstalled and reinstall and could see it scanned and added more domains. it means that this service does not scan frequently :(
